src/enterpriseCompatibilityTest/java/com/gradle/ccud/adapters/enterprise/BuildScanDataObfuscationAdapterTest.java

```diff
@@ -34,7 +33,7 @@ class BuildScanDataObfuscationAdapterTest {
     @BeforeEach
     void setup() {
         obfuscation = mock();
-        adapter = new BuildScanDataObfuscationAdapter(ProxyFactory.createProxy(obfuscation, BuildScanDataObfuscationProxy.class));
+        adapter = new BuildScanDataObfuscationAdapter(ProxyFactory.createProxy(obfuscation, BuildScanDataObfuscation.class));
```

Comment:
just for my understanding, why do we need the proxy here? can the adapter directly use the class directly?

Reply:
We don't necessarily use it in the test, but an adapter will be created in reality. So, I wanted tests to also go through proxy methods.